liteflow-labs/starter-kit
# Explore users page and ability to search users

### Project organization

- Trello card: https://trello.com/c/has1mFRK/434-enhance-user-discovery-and-search-explore-add-nfts-users-tabs
- Trello card: https://trello.com/c/CfsK467E/435-enhance-user-discovery-and-search-search-users-functionality
- closes: #108 

### Description

- Added a user page under a tab on the explore page
- Added ability to search users

### How to test

1. Visit the explore page
2. Notice the tabs NFT/Users
3. Click on the users tab
4. Notice list of users, use pagination to see if it works properly
5. Search a user. Click on users tab to see search results
6. When doing a new search under the subroute `explore` the route will not be updated and show the result on the current route. (e.g: searching a user while already on the `explore/users` page
